src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -138,24 +138,50 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
 
             return new ProgressStepDefinition[]
             {
+                // Some of the steps are broken into multiple sub-steps, either
+                // because the work needs to be done on a different thread or
+                // to report progress separate.
+
+                //*****************************************************************
+                // Initialization
+                //*****************************************************************
+                // Show an initial message and check the solution isn't dirty
```

Comment:
Miscellaneous: added documentation to explain the binding steps at higher level.